src/mumble/VolumeSliderWidgetAction.cpp

```diff
@@ -9,27 +9,26 @@
 #include <QSlider>
 #include <QToolTip>
 
-VolumeSliderWidgetAction::VolumeSliderWidgetAction(QObject *parent) : QWidgetAction(parent) {
-	m_volumeSlider = new QSlider(Qt::Horizontal);
-
-	m_volumeSlider->setMinimum(-30);
-	m_volumeSlider->setMaximum(30);
-	m_volumeSlider->setAccessibleName(tr("Slider for volume adjustment"));
-	m_volumeSlider->setSizePolicy(QSizePolicy::MinimumExpanding, QSizePolicy::MinimumExpanding);
-
-	connect(m_volumeSlider, &QSlider::valueChanged, this,
+VolumeSliderWidgetAction::VolumeSliderWidgetAction(QObject *parent)
+	: QWidgetAction(parent), m_volumeSlider(new QSlider(Qt::Horizontal), Mumble::QtUtils::deleteQObject) {
+	m_volumeSlider.get()->setMinimum(-30);
+	m_volumeSlider.get()->setMaximum(30);
+	m_volumeSlider.get()->setAccessibleName(tr("Slider for volume adjustment"));
+	m_volumeSlider.get()->setSizePolicy(QSizePolicy::MinimumExpanding, QSizePolicy::MinimumExpanding);
+
+	connect(m_volumeSlider.get(), &QSlider::valueChanged, this,
 			&VolumeSliderWidgetAction::on_VolumeSlider_valueChanged);
-	connect(m_volumeSlider, &QSlider::sliderReleased, this,
+	connect(m_volumeSlider.get(), &QSlider::sliderReleased, this,
 			&VolumeSliderWidgetAction::on_VolumeSlider_sliderReleased);
 
-	setDefaultWidget(m_volumeSlider);
+	setDefaultWidget(m_volumeSlider.get());
 
-	m_volumeSlider->setProperty("mouseTracking", true);
+	m_volumeSlider.get()->setProperty("mouseTracking", true);
 }
 
 void VolumeSliderWidgetAction::updateSliderValue(float value) {
 	int dbShift = VolumeAdjustment::toIntegerDBAdjustment(value);
-	m_volumeSlider->setValue(dbShift);
+	m_volumeSlider.get()->setValue(dbShift);
```

Comment:
No need to `get` when using the `->` operator. This operator is overloaded for smart pointers to do the correct thing.